src/components/Reports/FourteenMonthReports/handleFourteenMonthReportData.ts

```diff
@@ -0,0 +1,148 @@
+import { FourteenMonthReport } from 'pages/api/graphql-rest.page.generated';
```

Comment:
Could we import this from the datahandler? Or did you have to customize some things?

Reply:
Oh sorry, I meant can we import everything in this file, instead of copying it?